src/stores/UsersStore.js

```diff
@@ -18,6 +15,22 @@ class UserStore {
   deleteUser() {
     this.currentUser = {};
   }
+
+  @computed get getPictureAvatarLink() {
+    return this.currentUser.avatar;
+  }
+
+  setToUsersList(data){
+    this.usersList.push(data);
+  }
+
+  deleteUsersList() {
+    this.usersList = [];
+  }
+
+  @computed get getFullName() {
```

Comment:
You can remove this and use `UserStore.currentUser.fullName`. If you still want to have this in store, you can `return this.currentUser.fullName` instead